### PR TITLE
Allow a level to inherit configuration from multiple levels.

### PR DESCRIPTION
**Pull request by tdenewiler**

### CODE REVIEW

File: statick_tool/plugins/tool/xmllint_tool_plugin.py [hunk 1]
```diff
@@ -21,9 +21,13 @@ def scan(self, package: Package, level: str) -> Optional[List[Issue]]:
         flags: List[str] = []
         flags += self.get_user_flags(level)
 
+        files: List[str] = []
```
Comment (xydesa) [inline statick_tool/plugins/tool/xmllint_tool_plugin.py:24]: didn't these changes get merged in #403? how come they're still listed here

File: tests/config/rsc/user.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 levels:
   custom:
-    inherits_from: "threshold"
+    inherits_from:
```
Comment (xydesa) [inline tests/config/rsc/user.yaml:3]: what about testing inheriting from levels in both the base config file and the custom file?

File: statick_tool/plugins/tool/xmllint_tool_plugin.py (hunk 1 above)
Comment (tdenewiler) [reply statick_tool/plugins/tool/xmllint_tool_plugin.py]: I merged upstream (sscpac) main into my source branch and cleaned that up.

**Review by gregtkogut — APPROVED** [review]

File: tests/config/rsc/user.yaml (hunk 2 above)
Comment (xydesa) [reply tests/config/rsc/user.yaml:3]: that tests for multiple levels from one file, but not one level from the custom config and one level from the base statick config.

Comment (tdenewiler) [reply tests/config/rsc/user.yaml:3]: [b7ee16e](https://github.com/sscpac/statick/pull/405/commits/b7ee16ec5c1d6e31a0c4f2bda83656cfac839111). This test uses a base configuration file and a user configuration file. The level we are testing inherits from levels in both base and user configuration files. I had to make the test `config.yaml` have reporting plugins to test that part, where the addition matches the main Statick configuration file.

**Review by xydesa — APPROVED** [review]

**Pull request merged**